packages/tenderly-actions/src/actions/networkConfig.ts

```diff
@@ -0,0 +1,421 @@
+  },
+  [Network.APECHAIN]: {
+    isSupported: false,
+  },
+  [Network.CURTIS]: {
+    isSupported: false,
+  },
+  [Network.KINTO]: {
+    isSupported: false,
+  },
+  [Network.BERACHAIN]: {
+    isSupported: false,
+  },
+  [Network.BERACHAIN_BARTIO]: {
+    isSupported: false,
+  },
+  [Network.BEPOLIA]: {
+    isSupported: false,
+  },
+  [Network.CONCRETE]: {
+    isSupported: false,
+  },
+  [Network.INK_SEPOLIA]: {
+    isSupported: false,
+  },
+  [Network.BLE_TESTNET]: {
+    isSupported: false,
+  },
+  [Network.SEI_PACIFIC_1]: {
+    isSupported: false,
+  },
+  [Network.SEI_ATLANTIC_2]: {
+    isSupported: false,
+  },
+  [Network.SONEIUM]: {
+    isSupported: false,
+  },
+  [Network.SONEIUM_MINATO]: {
+    isSupported: false,
+  },
+  [Network.METIS_ANDROMEDA]: {
+    isSupported: false,
+  },
+  [Network.METIS_SEPOLIA]: {
+    isSupported: false,
+  },
+  [Network.FLARE]: {
+    isSupported: false,
+  },
+  [Network.SWELLCHAIN]: {
+    isSupported: false,
+  },
+  [Network.SWELLCHAIN_SEPOLIA]: {
+    isSupported: false,
+  },
+  [Network.CORN]: {
+    isSupported: false,
+  },
+  [Network.CORN_TESTNET]: {
+    isSupported: false,
+  },
+  [Network.SONIC]: {
+    isSupported: false,
+  },
+  [Network.SONIC_BLAZE]: {
+    isSupported: false,
+  },
+  [Network.SCROLL_SEPOLIA]: {
+    isSupported: false,
+  },
+  [Network.SCROLL]: {
+    isSupported: false,
+  },
+  [Network.RONIN]: {
+    isSupported: false,
+  },
+  [Network.RONIN_TESTNET]: {
+    isSupported: false,
+  },
+  [Network.UNICHAIN]: {
+    isSupported: false,
+  },
+  [Network.ZKSYNC]: {
+    isSupported: false,
+  },
+  [Network.SOPHON]: {
+    isSupported: false,
+  },
+  [Network.SOPHON_TESTNET]: {
+    isSupported: false,
+  },
+  [Network.STORY]: {
+    isSupported: false,
+  },
+  [Network.STORY_AENEID]: {
+    isSupported: false,
+  },
+  [Network.LENS]: {
+    isSupported: false,
+  },
+  [Network.LENS_SEPOLIA]: {
+    isSupported: false,
+  },
+  [Network.HOODI]: {
+    isSupported: false,
+  },
+  [Network.KATANA_TATARA]: {
+    isSupported: false,
+  },
+  [Network.AF_NEXUS_TESTNET]: {
+    isSupported: false,
+  },
+  [Network.ZIRCUIT]: {
+    isSupported: false,
+  },
+  [Network.ZIRCUIT_GARFIELD_TESTNET]: {
+    isSupported: false,
+  },
+  [Network.INJECTIVE_TESTNET]: {
+    isSupported: false,
+  },
+  [Network.CELO_ALFAJORES_TESTNET]: {
+    isSupported: false,
+  },
+  [Network.CELO]: {
+    isSupported: false,
+  },
+  [Network.CONVERGE_TESTNET]: {
+    isSupported: false,
+  },
+  [Network.OMEGA_10]: {
+    isSupported: false,
+  },
+  [Network.TAC]: {
+    isSupported: false,
+  },
+  [Network["TAC_SPB_TESTNET"]]: {
+    isSupported: false,
+  },
+  [Network.SHAPE]: {
+    isSupported: false,
+  },
+  [Network["SHAPE_SEPOLIA"]]: {
+    isSupported: false,
+  },
+} as const;
+
+export type SingleNetworkConfig = NetworkConfig[keyof NetworkConfig];
+
+export const getNetworkConfig = (network: keyof NetworkConfig | undefined): SingleNetworkConfig => {
```

Comment:
```suggestion
export const getNetworkConfig = (network: keyof NetworkConfig | undefined): SingleNetworkConfig | undefined => {
```

Now TS will warn us we have to treat the fact that the network config might not be found downstream...